server/index.js

```diff
@@ -5,7 +5,19 @@ const port = 7777;
 const db = require("../database/index.js");
 
 app.use(express.static(path.join(__dirname, "../public/dist")));
-
 app.use(express.json());
 
+app.get("/5", (req, res) => {
```

Comment:
Is this route (/5) for testing?

Reply:
Yeah, that's for testing. I hard-coded a few things before knowing how the proxy routing will look